logistics_project/apps/malawi/extensions/logistics/supplypoint.py

```diff
@@ -0,0 +1,33 @@
+from django.db import models
+from logistics.models import Product
+from logistics.util import config
+
+
+class MalawiSupplyPointExtension(models.Model):
```

Comment:
yeah the main reason was to avoid modifying core code. (2) actually wouldn't be possible without modifying core code either because commodities_stocked() is referenced in other places in rapidsms-logistics. updated comment here: https://github.com/dimagi/logistics/pull/261